pkg/hhfctl/inspect/inspect.go

```diff
@@ -151,20 +153,45 @@ func Render[TOut Out](now time.Time, output OutputType, w io.Writer, out TOut) e
 func RenderTable(headers []string, data [][]string) string {
 	str := &strings.Builder{}
 
-	table := tablewriter.NewWriter(str)
-	table.SetHeader(headers)
-	table.SetAutoWrapText(true)
-	table.SetAutoFormatHeaders(true)
-	table.SetHeaderAlignment(tablewriter.ALIGN_LEFT)
-	table.SetAlignment(tablewriter.ALIGN_LEFT)
-	table.SetCenterSeparator("")
-	table.SetColumnSeparator("")
-	table.SetRowSeparator("")
-	table.SetHeaderLine(false)
-	table.SetTablePadding("\t")
-	table.SetNoWhiteSpace(true)
-	table.AppendBulk(data)
-	table.Render()
+	cfg := tablewriter.Config{
+		Row: tw.CellConfig{
+			Formatting: tw.CellFormatting{
+				AutoWrap:  tw.WrapNormal,
+				Alignment: tw.AlignLeft,
+			},
+			Padding: tw.CellPadding{Global: tw.Padding{Right: "    "}},
+		},
+		Header: tw.CellConfig{
+			Formatting: tw.CellFormatting{
+				AutoWrap:  tw.WrapNormal,
+				Alignment: tw.AlignLeft,
+			},
+			Padding: tw.CellPadding{Global: tw.Padding{Right: "    "}},
+		},
+	}
+	rendition := tw.Rendition{
+		Borders: tw.BorderNone,
+		Settings: tw.Settings{
+			Lines:      tw.LinesNone,
+			Separators: tw.SeparatorsNone,
+		},
+	}
+
+	table := tablewriter.NewTable(str,
+		tablewriter.WithRenderer(renderer.NewBlueprint(rendition)),
+		tablewriter.WithConfig(cfg),
+	)
+	table.Header(headers)
+	err := table.Bulk(data)
+	if err != nil {
+		slog.Error("Error in adding bulk data to table", "error", err)
+		return "Error"
+	}
+	err = table.Render()
+	if err != nil {
+		slog.Error("Error in table rendering", "error", err)
+		return "Error"
```

Comment:
Probably not the best error handling here. My thought process was, log the error and put in some error text in the hope that it is only partial problem, so for example the port speeds mess up somehow and they will show "Error" but the other info could be okay.